src/lib/tradeData.ts

```diff
@@ -141,13 +141,13 @@ export const getTradeInputAlert = ({
 
 export const calculateCrossPositionMargin = ({
   notionalTotal,
-  adjustedMmf,
+  adjustedImf,
 }: {
   notionalTotal?: Nullable<number>;
-  adjustedMmf?: Nullable<number>;
+  adjustedImf?: Nullable<number>;
 }) => {
   const notionalTotalBN = MustBigNumber(notionalTotal);
-  const adjustedMmfBN = MustBigNumber(adjustedMmf);
+  const adjustedMmfBN = MustBigNumber(adjustedImf);
```

Comment:
adjustedMmfBN -> adjustedImfBN

Also does this affect the output of the cross position margin since we actually do want to use mmf here?

Reply:
I think for cross position margin we DO want imf, that's my assertion in this PR.